target/linux/rockchip/armv8/config-5.4

```diff
@@ -323,11 +323,12 @@ CONFIG_PHYLIB=y
 CONFIG_PHYLINK=y
```

Comment:
```config-5.4``` is look weird, out of sync?

Reply:
not ready yet, I want to sort out the sound and crypto, I have hdmi fbdev working

Reply:
I'll leave sound and crypto for a later PR if not ready